src/Psl/Type/backed_enum_value.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Psl\Type;
+
+use BackedEnum;
+
+/**
+ * @psalm-pure
+ *
+ * @template T of BackedEnum
+ *
+ * @param class-string<T> $enum
+ *
+ * @return TypeInterface<value-of<T>>
+ */
+function backed_enum_value(string $enum): TypeInterface
```

Comment:
You can document this new function in https://github.com/azjezz/psl/blob/next/src/Psl/Type/README.md